app/src/crypto.c

```diff
@@ -172,10 +163,9 @@ zxerr_t crypto_sign(uint8_t *output,
     return error;
 }
 
-void ripemd160_32(uint8_t *out, uint8_t *in) {
-    cx_ripemd160_t rip160;
-    cx_ripemd160_init(&rip160);
-    cx_hash_no_throw(&rip160.header, CX_LAST, in, CX_SHA256_SIZE, out, CX_RIPEMD160_SIZE);
+static zxerr_t ripemd160_32(uint8_t *out, uint8_t *in) {
```

Comment:
I'm not sure there is a reason for such a function now. Could be called directly I guess.